libbindgen/tests/expectations/tests/anon_union.rs

```diff
@@ -42,18 +42,8 @@ pub const TErrorResult_UnionState_HasException: TErrorResult_UnionState =
 #[repr(i32)]
 #[derive(Debug, Copy, Clone, PartialEq, Eq, Hash)]
 pub enum TErrorResult_UnionState { HasMessage = 0, }
-#[repr(C)]
-#[derive(Debug, Copy, Clone)]
-pub struct TErrorResult_Message<T> {
-    pub _address: u8,
-    pub _phantom_0: ::std::marker::PhantomData<T>,
-}
-#[repr(C)]
-#[derive(Debug, Copy, Clone)]
-pub struct TErrorResult_DOMExceptionInfo<T> {
-    pub _address: u8,
-    pub _phantom_0: ::std::marker::PhantomData<T>,
-}
+pub enum TErrorResult_Message { }
```

Comment:
This is wrong, you still need to make it take a template parameter, there is no way this is compiling otherwise.